cups/ppd-util.c

```diff
@@ -561,8 +559,7 @@ cups_get_printer_uri(
 		  "printer-uri-supported"
 		};
 
-
-  DEBUG_printf(("4cups_get_printer_uri(http=%p, name=\"%s\", host=%p, hostsize=%d, resource=%p, resourcesize=%d, depth=%d)", http, name, host, hostsize, resource, resourcesize, depth));
+  DEBUG_printf(("4cups_get_printer_uri(http=%p, name=\"%s\", host=%p, hostsize=%d, resource=%p, resourcesize=%d)", http, name, host, hostsize, resource, resourcesize));
```

Comment:
The convention we've been using is two blank lines after variables in the beginning of a function, one when the variables are in an embedded scope within a function.